Fastlane/deployment_lanes.rb

```diff
@@ -325,6 +325,12 @@
   UI.message "IPA saved at #{ENV['IPA_OUTPUT_PATH']}"
 end
 
+desc 'Just install the certs onto bitrise so this can build for test lab'
+lane :install_certs do
+  clear_derived_data
```

Comment:
Why do we need this?

Reply:
Thought that would assure a clean slate in bitwise when we do a rebuild of the failed workflow. Can remove this if not required.

Reply:
Hm, I'm not entirely sure, but I think we either already have this happen somewhere else, or we always have a clean snapshot when retrying builds. Perhaps @neodym knows more? 😅

Reply:
removed